### ISSUE TITLE
Map pins

### ISSUE THREAD

**Issue by xVilho** [SpikeHimself/XPortal#101]
I love your mod, it makes portals so much nicer to use. Though, am bit sick of manually pinning portals to map. There is few mods that are supposed to do that, but they are broken and i think the devs are inactive.

Got few ideas how pinning could be done:
1. Automatically when nametag is updated
2. When pressing PING button
3. Add refresh pins button with like 10sec CD

Maybe in later versions could add custom icons that would be easier to see on the map with lots of vanilla pins.

**Comment by SpikeHimself** [comment]
Hi @xVilho, thank you for your suggestion! 

I think this is pretty similar to #71?

**Comment by xVilho** [comment]
Yes sounds similar. Sorry for not checking if there isnt similar tickets. 
I found out that HUDcompass by Neobotics does automatic pinning for portals. Though the pins are small and use the default pin icon.
Whatever you decide thank you for answering and have nice day ^^
